sim/side.ts

```diff
@@ -445,12 +445,16 @@ export class Side {
 
 		const lockedMove = pokemon.getLockedMove();
 		if (lockedMove) {
-			const lockedMoveTarget = pokemon.lastMoveTargetLoc || 0;
```

Comment:
I remember you were asking me how this worked, and I couldn't figure it out, either. I'm glad you figured it out!

(I think this should probably be documented in the `lockedmove` and `twoturnmove` conditions in `conditions.ts`.